lib/pages/forms_pages/adult_report_page.dart

```diff
@@ -75,7 +75,7 @@ class _AdultReportPageState extends State<AdultReportPage> {
     _pagesController = PageController();
     index = 0.0;
     _initialformsRepot = [
-      AddPhotoButton(true, true, _checkAtLeastOnePhotoAttached),
+      AddPhotoButton(true, true, _checkAtLeastOnePhotoAttached, 'adult'),
```

Comment:
Shouldn't be 'adult' replaced with `MyLocalizations.of(context, 'one_mosquito_reminder_badge')` here so AddPhotoButton class is agnostic of the report type?

Reply:
If modified it to be type agnostic. Same as the other PR, I have to pass the translation key instead of the value directly, and then from `AppPhotoButton` again pass it to `WhatsappCamera` to build the widget